components/dashboard/balance.tsx

```diff
@@ -0,0 +1,31 @@
+type BalanceProperty = {
```

Comment:
Idem pour Props

Reply:
fixed in: https://github.com/marigold-dev/tzsafe-ui/pull/167/commits/34232e4aa6aed7278eb8921d2554a358cf208fca